src/features/home/courses/courses.tsx

```diff
@@ -1,96 +1,60 @@
-import { ReactNode } from 'react';
+import { formatCoursesData } from './lib/formatCoursesData';
+import { getCourseIcon } from './lib/getCourseIcon';
+import { IconsTitle } from './lib/icons.data';
+import { CourseCard } from './ui/CourseCard';
 import { Button } from '@/app/components';
-import { ROUTES } from '@/app/const';
-import { useWindowSize } from '@/app/hooks';
-import { buildUrl } from '@/app/services/platform';
-import { AngularIcon, AwsLogo, HtmlIcon, ReactIcon, RsBanner } from '@/icons';
+import { useDataByName, useWindowSize } from '@/app/hooks';
+import { Course } from '@/app/types';
+import { RsBanner } from '@/icons';
 
 import './courses.scss';
 
-type CourseProps = {
-  title: string;
-  language: string;
-  startDate: string;
-  href: string;
-  icon: ReactNode;
-};
-
-const courses: CourseProps[] = [
-  {
-    title: 'AWS Fundamentals',
-    language: 'EN',
-    startDate: 'April 15, 2024',
-    href: buildUrl(`/${ROUTES.COURSES}/${ROUTES.AWS_FUNDAMENTALS}`),
-    icon: <AwsLogo />,
-  },
-  {
-    title: 'AWS Cloud Developer',
-    language: 'EN',
-    startDate: 'May 28, 2024',
-    href: buildUrl(`/${ROUTES.COURSES}/${ROUTES.AWS_DEVELOPER}`),
-    icon: <AwsLogo />,
-  },
-  {
-    title: 'JS / Front‑end. Pre‑school',
-    language: 'RU',
-    startDate: 'June 24, 2024',
-    href: buildUrl(`/${ROUTES.COURSES}/${ROUTES.JS_PRESCHOOL_RU}`),
-    icon: <HtmlIcon />,
-  },
-  {
-    title: 'React',
-    language: 'EN',
-    startDate: 'July 1, 2024',
-    href: buildUrl(`/${ROUTES.COURSES}/${ROUTES.REACT}`),
-    icon: <ReactIcon />,
-  },
-  {
-    title: 'Angular',
-    language: 'EN',
-    startDate: 'July 1, 2024',
-    href: buildUrl(`/${ROUTES.COURSES}/${ROUTES.ANGULAR}`),
-    icon: <AngularIcon />,
-  },
-];
-
 export const Courses = () => {
   const size = useWindowSize();
+  const laptopScreenBreakPoint = 1440;
+  const tabletScreenBreakPoint = 810;
```

Comment:
You can use ready constants
![image](https://github.com/rolling-scopes/site/assets/79564962/9e8bebe7-3e6c-4887-9fd8-78e298f57cdd)

Reply:
Do we really need to install additional libraries just to extract 2 variables from scss file?

Reply:
I was thinking the same thing. So let's close this comment